tests/test_ensembles.py

```diff
@@ -10,11 +10,17 @@
     import xesmf as xe
 except ImportError:
     xe = None
+# temp fix for changes to xclim-testdata
+from functools import partial
+
 from xclim.testing import open_dataset
 from xclim.testing.helpers import test_timeseries as timeseries
 
 import xscen as xs
 
+open_dataset = partial(open_dataset, branch="old-testdata")
```

Comment:
You could use the last tagged version: https://github.com/Ouranosinc/xclim-testdata/releases/tag/v2023.12.14

That's what they're there for!

Reply:
Once my work in `xclim` and `clisops` is done, I'll post the changes to use pooch in a new PR. The new system is relatively simpler (and can even be much more simplified if we can accept a few breaking changes).

Reply:
Does `open_dataset` support passing a tag ? I thought not!

Also, once we have a release of xclim with your changes, I don't think there will be a need for any changes here! We simply call `open_dataset`, as long as this one works, we're good.

Reply:
That's true, you should be able to base things directly off of `xclim` going forward.

Since a tag in `git` is treated like a branch pointing to a specific commit, the `branch` call signature accepts a tag name! The same will be true going forward with the new system.